app/portainer/views/sidebar/sidebarController.js

```diff
@@ -1,36 +1,37 @@
 angular.module('portainer.app')
```

Comment:
Anything was changed here beside code style? If it's only style I don't think it should be in this PR

Reply:
Agree, please avoid the use of formatting tools to make it easier to review.